src/main/java/jenkins/plugins/slack/pipeline/SlackUploadFileStep.java

```diff
@@ -182,17 +181,4 @@ private static String getThreadTs(String channelName) {
         }
         return null;
     }
-
-    private static String cleanChannelName(String channelName) {
-        String[] splitForThread = channelName.split(":", 2);
-        String channel = channelName;
-        if (splitForThread.length == 2) {
-            channel = splitForThread[0];
-            if (channel.startsWith("#")) {
```

Comment:
previously was only removed if a thread_ts was attached which was probably more common in pipeline but not always there.